packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx

```diff
@@ -0,0 +1,146 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Box,
+  ButtonLink,
+  Card,
+  IconButton,
+  Tag,
+  Text,
+  Icon,
+} from '@fuel-ui/react';
+import { motion } from 'framer-motion';
+import type { FC } from 'react';
+
+import type { Project } from '../../types';
+import { STATUS_TEXT } from '../../types';
+
+import { ProjectItemLoader } from './ProjectItemLoader';
+
+import { animations } from '~/systems/Core';
+
+const MotionCard = motion(Card);
+
+export type ProjectItemProps = Project & {
+  onPress?: () => void;
+};
+
+type ProjectItemComponent = FC<ProjectItemProps> & {
+  Loader: typeof ProjectItemLoader;
+};
+
+export const ProjectItem: ProjectItemComponent = ({
+  name,
+  description,
+  url,
+  status,
+}: ProjectItemProps) => {
+  let hostname = '';
+  try {
+    hostname = new URL(url || 'https://#').hostname;
+  } catch (e) {
+    hostname = '';
+  }
+
+  const onCardPress = () => {
+    window.open(url, '_blank');
+  };
+
+  return (
+    <MotionCard
+      withDividers
+      css={styles.card}
+      {...animations.slideInBottom({
+        transition: { type: 'spring' },
+      })}
+      onClick={onCardPress}
+    >
+      <Card.Body>
```

Comment:
this <Card.Body> padding should be:
```
py: '$2'
```

Reply:
Not really, according to the design, the padding is the same on all side.

<img width="1081" alt="Screenshot 2023-06-23 at 2 09 24 PM" src="https://github.com/FuelLabs/fuels-portal/assets/15849997/93151438-e7c2-40b3-85fb-a133bad85209">

Reply:
I removed the padding on the external Card component, and all the padding on the Card.Body is equal.